app/mailers/thredded/private_topic_mailer.rb

```diff
@@ -1,8 +1,9 @@
 # frozen_string_literal: true
 module Thredded
   class PrivateTopicMailer < Thredded::BaseMailer
-    def message_notification(private_topic_id, emails)
+    def message_notification(private_topic_id, post, emails)
```

Comment:
Let's pass `post_id` here and look it up like this:

```ruby
@post = find_record Thredded::PrivatePost, post_id
```

This avoids serializing the entire record into job arguments and also ensures the latest version of the post is used when job is executed.